### PR TITLE
Abelian ∞-groups

### PR DESCRIPTION
**Pull request by fredrik-bakke**
Defines abelian ∞-groups as ∞-groups that are $n$-deloopable for all $n$. In other words, there is a connective spectrum where the group appears as the first type in the sequence.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
After spending most of the evening on this, I just now came over [Higher Groups in Homotopy Type Theory](https://arxiv.org/pdf/1802.04315) and see there's a nice easily formalizable hierarchy of increasingly coherent commutative  ∞-groups. I wouldn't want to mess up the infrastructure by having this suboptimal formalization merged. I'll see if I can find the time to do this more properly one of the coming days instead.

**Comment by fredrik-bakke** [comment]
It's crazy how fast 3 weeks pass sometimes...

**Comment by fredrik-bakke** [comment]
So that this PR doesn't die, I've wrapped it up with an added caveat that the formalization is one of multiple possible approaches.

File: src/synthetic-homotopy-theory/connective-prespectra.lagda.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+open import synthetic-homotopy-theory.suspensions-of-types
+```
+
+</details>
+
+## Idea
+
+A [prespectrum](synthetic-homotopy-theory.prespectra.md) is
+{{#concept "connective" Disambiguation="prespectrum" Agda=is-connective-Prespectrum}}
+if the $n$'th type in the [sequence](foundation.sequences.md) is
```
Comment (EgbertRijke) [inline src/synthetic-homotopy-theory/connective-prespectra.lagda.md:37]: Apparently the American spelling for nth is nth without hyphen and without apostrophe. I'm not sure if n'th is ever correct. 

```suggestion
if the $n$th type in the [sequence](foundation.sequences.md) is
```

File: src/synthetic-homotopy-theory/connective-spectra.lagda.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+</details>
+
+## Idea
+
+A [spectrum](synthetic-homotopy-theory.spectra.md) is
+{{#concept "connective" Disambiguation="spectrum" Agda=is-connective-Spectrum}}
+if the underlying [prespectrum](synthetic-homotopy-theory.prespectra.md) is
+[connective](synthetic-homotopy-theory.connective-prespectra.md). I.e., if the
+$n$'th type in the [sequence](foundation.sequences.md) is
```
Comment (EgbertRijke) [inline src/synthetic-homotopy-theory/connective-spectra.lagda.md:41]:
```suggestion
$n$th type in the [sequence](foundation.sequences.md) is
```

**Review by EgbertRijke — APPROVED** [review]
Nice pull request!

**Pull request merged**